html/modules/custom/hpc_api/src/Query/EndpointPlanQuery.php

```diff
@@ -200,4 +208,30 @@ public function getFlowSourcesByPlan($plan_id) {
     return $sources;
   }
 
+  /**
+   * Get plan icon.
+   */
+  public function getPlanIcon($plan_id) {
```

Comment:
There are no plan icons in the API. The only icons there are and that Insights cares about are GVE (cluster) icons, see https://hum-insight.info/plan/1037/ge/6030 for an example. But those are already in a sub navigation element, so I think they are not needed here.